wordpress/pwnscriptum/README.md

```diff
@@ -1,23 +1,23 @@
-# Wordpress 4.6 任意命令执行漏洞（PwnScriptum）
+# Wordpress 4.6 Remote Code Execution Vulnerability（PwnScriptum）
 
-## 原理
+[中文版本(Chinese version)](README.zh-cn.md)
 
-参考 https://exploitbox.io/vuln/WordPress-Exploit-4-6-RCE-CODE-EXEC-CVE-2016-10033.html
+## Reference link
 
-## 测试环境
+https://exploitbox.io/vuln/WordPress-Exploit-4-6-RCE-CODE-EXEC-CVE-2016-10033.html
 
-编译及运行测试环境
+## Environment setup
 
 ```
 docker-compose build
 docker-compose up -d
 ```
 
-由于Mysql初始化需要一段时间，所以请等待。成功运行后，打开站点，初始化管理员用户名和密码后即可使用（数据库等已经配置好，且不会自动更新）。
+Because Mysql initialization need a little time, so please wait. After initialization success, visit the site, set the administrator username and password to use it (the database is configured and not automatically updated).
```

Comment:
needs

Reply:
After initialization `success` -> After initialization `succeeds` or `is complete`